manuscript/100_Analysis_In_TDD.md

```diff
@@ -109,7 +109,7 @@ TODO list... again!
 
 As we said previously, a�TODO list is a�repository for anything that comes to our mind when writing or thinking about a�Statement, but is not a�part o the current Statement we are writing. We do not want to forget it, neither do we want it to haunt us and distract us from our current task, so we write it down as soon as possible and continue with our current task.
 
-Suppose you are writing a�piece of small logic that allows user access when he is an employee of a�zoo, but denies access if he is a�guest of the zoo. Then, after starting writing a�Statement it gets to you that actually any employee can be a�guest as well -- for example, he might choose to visit the zoo with his family during his vacation. Still, the two previous rules hold, so not to allow this case to distract you, you quickly add an item to the TODO list (like �TODO: what if someone is an employee, but comes to the zoo as a�guest?") and finish the current Statement. When you are finished, you can always come back to the list and pick item to do next.
+Suppose you are writing a�piece of small logic that allows user access when he is an employee of a�zoo, but denies access if he is a�guest of the zoo. Then, after starting writing a�Statement it gets to you that any employee can be a�guest as well -- for example, he might choose to visit the zoo with his family during his vacation. Still, the two previous rules hold, so not to allow this case to distract you, you quickly add an item to the TODO list (like �TODO: what if someone is an employee, but comes to the zoo as a�guest?") and finish the current Statement. When you are finished, you can always come back to the list and pick item to do next.
```

Comment:
It's as with code, when you return to it a couple of times you likely spot opportunities for improvement.